src/dlgAboutDialog.cpp

```diff
@@ -1062,3 +1072,17 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
     textBrowser_supporters->setDocument(supportersDocument.get());
     textBrowser_supporters->setOpenExternalLinks(true);
 }
+
+QString dlgAboutDialog::createBuildInfo() const {
+
+    QString commitHash = !QByteArray(GIT_COMMIT_HASH).isEmpty() ? QStringLiteral(GIT_COMMIT_HASH) : QUESTION_MARK;
+    QString branch = !QByteArray(GIT_BRANCH).isEmpty() ? QStringLiteral(GIT_BRANCH) : QUESTION_MARK;
+    return QStringLiteral("<table border=\"0\" style=\"margin-bottom:36px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n")
+        .append(QStringLiteral("<tr><td colspan=\"2\" style=\"font-style:italic;\">%1<br></td></tr>").arg(tr("It would be useful to copy and include information below when reporting an issue!")))
+        .append(QStringLiteral("<tr><td style=\"font-weight: 800\">%1</td><td>%2%3</td></tr>").arg(tr("Build information:"), tr("Build: "), mudlet::self()->version))
```

Comment:
What is a build to the coders doesn't make sense to the non-coders:

```suggestion
        .append(QStringLiteral("<tr><td style=\"font-weight: 800\">%1</td><td>%2%3</td></tr>").arg(tr("Technical information:"), tr("Version: "), mudlet::self()->version))
```

Reply:
:+1: Fair comment...